app/lib/logstash_logging.rb

```diff
@@ -1,5 +1,6 @@
 class LogstashLogging
   DOMAIN_FOR_LOGS = Rails.env.production? ? AzureEnvironment.hostname : Socket.gethostname
+  SERVICE_NAME = ENV['SERVICE_NAME'] # e.g. web, worker or clock
```

Comment:
I still think 'service' fits, because all major cloud providers seem to use 'service' when talking about stack components that can be scaled or load-balanced independently. It seems to me that each of these container types should be defined in Azure in a way that lets them scale independently. What are your thoughts @tpinney?

But yes, SERVICE_NAME is a bit misleading. How about SERVICE_TYPE instead?